pkg/actuators/machine/reconciler.go

```diff
@@ -189,7 +189,11 @@ func (r *Reconciler) reconcileMachineWithCloudState(conditionFailed *ibmcloudpro
 	}
 
 	clusterID := r.machine.Labels[machinev1.MachineClusterIDLabel]
-	providerID := fmt.Sprintf("ibmvpc://%s/%s/%s", clusterID, r.providerSpec.Zone, r.machine.GetName())
+	accountID, err := r.ibmClient.GetAccountID()
+	if err != nil {
+		return fmt.Errorf("get instance failed with an error: %q", err)
+	}
+	providerID := fmt.Sprintf("ibm://%s///%s/%s", accountID, clusterID, *newInstance.ID)
```

Comment:
Can we copy the comment from the CCM which describes the format of this, and maybe link to the CCM code as well?

Reply:
Sure, I will do that

Reply:
added